test/get-current-committish.ts

```diff
@@ -0,0 +1,95 @@
+import test from 'ava';
```

Comment:
It's best to move these tests back to tests/helpers.ts. It's hard to see what's being changed at all

Reply:
👍